build.sbt

```diff
@@ -3,7 +3,7 @@ import sbt.Def
 import scoverage.ScoverageKeys
 import uk.gov.hmrc.versioning.SbtGitVersioning.autoImport.majorVersion
 
-lazy val appName: String = "emcs-tfe-report-a-receipt-frontend"
+lazy val appName: String = "emcs/report-receipt"
```

Comment:
This shouldn't have changed, looks like it got caught by a global find and replace by mistake - whoops 🚀